### PR TITLE
SQLite: Supporting Comdb2 data types in the JSON1 extension.

### PR DESCRIPTION
**Pull request by riverszhang89**

### CODE REVIEW

File: tests/jsonify.test/runit [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+bash -n "$0" | exit 1
+
+dbnm=$1
+
+set -e
+
+# Make sure that all queries go to the same node.
```
Comment (nirbhayc) [inline tests/jsonify.test/runit]: Why?

File: tests/jsonify.test/runit [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+set +e
+diff  <(echo "$expected" ) <(echo "$actual")
+if [ "$?" != "0" ]; then
+  echo 'Failed.'
+  exit 1
+fi
+
+echo 'Passed.'
+exit 0
```
Comment (nirbhayc) [inline tests/jsonify.test/runit]: I'd have used the xxx.sql/xxx.expected approach, makes it easier to add more cases.

File: tests/jsonify.test/runit (hunk 1 above)
Comment (riverszhang89) [reply tests/jsonify.test/runit]: Sorry. It is a copy/paste error. I will remove.

File: tests/jsonify.test/json_object.sql.expected [hunk 3]
```diff
@@ -0,0 +1,2 @@
+1
+{"dtms":2017-10-17T141720.649 America/New_York,"dtus":2017-10-17T141720.648885 America/New_York,"intvym":10-00,"intvds":365 00:00:00.123,"intvdsus":365 00:00:00.123456,"dec":3.141593}
```
Comment (nirbhayc) [inline tests/jsonify.test/json_object.sql.expected]: Not valid JSON. The values need to be quoted.

**Pull request merged**